.github/workflows/build-citus-community-nightlies.yml

```
@@ -1,4 +1,4 @@
-name: Buildand publish citus nightly package
+name: Build and publish citus nightly packages
```

very minor, to distinguish from enterprise one:

```suggestion
name: Build and publish citus community nightly packages
```

---

Fixed